### DISCUSSION TITLE
Monetizing with dynamic advertising?

### DISCUSSION THREAD

**Discussion by onorosaurosrex** [advplyr/audiobookshelf#913] [Ideas]
Hi, I just discovered this repository, and have this question.
Is there a way to dynamically add a small track working as a intro to any chapter played starting and at the for it to work as an advertising source. So we can change the audio comercial without touching the files and be able to schedule dates or times it sounds?

**Comment by advplyr** [comment]
No, I think you need to look for a completely different software. That is not even in our ballpark.
